src/backend/db/categories.js

```diff
@@ -8,20 +8,33 @@ import { v4 as uuid } from "uuid";
 export const categories = [
   {
     _id: uuid(),
-    categoryName: "Computer Programming",
+    category: "photography",
     description:
       "Computer programming is the process of designing and building an executable computer program to accomplish a specific computing",
   },
   {
     _id: uuid(),
-    categoryName: "Frontend Development",
+    category: "communication",
     description:
```

Comment:
Capitalize the first letter of the category names.It doesn't look good in the UI.Other than that code looks clean and good to merge.